src/test/java/com/codurance/lightaccess/PGOneToManyShould.java

```diff
@@ -0,0 +1,84 @@
+package com.codurance.lightaccess;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
+
+import java.util.*;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+@RunWith(Parameterized.class)
+public class PGOneToManyShould {
+
+
+    private Map<Integer, List<Optional>> actual;
+    private List<Tuple> expected;
+
+    public PGOneToManyShould(Map<Integer, List<Optional>> actual, List<Tuple> expected) {
+        this.actual = actual;
+        this.expected = expected;
+    }
+
+    @Test
+    public void putAnEmptyValueAndNotStoreAnyValue() throws Exception {
+        PGOneToMany<Integer, Integer> pgOneToMany =

```

Comment:
When I look at the code of the test it is hard to find out what is the role of the collect method because: 
- it is parametrised
- lambda passed to collect is not described
- use magic number (`a`, `b`)